components/layout/Header.tsx

```diff
@@ -63,7 +63,6 @@ export default function Header() {
   return (
     <>
       {/* hide until we figure out what the cta is supposed to be */}
```

Comment:
@GabrielEger2 When you do the PR to revert giving tuesday, will you also delete this commented out line, and the banner import? Thanks.

Reply:
sure